### PR TITLE
chore: lift can wrap 1-arity function

### PR DESCRIPTION
**Pull request by adispring**

### CODE REVIEW

**Comment by adispring** [comment]
When I run unit test, `R.trim` throw an test error, as follows: 

```js
1109 passing (4s)
  1 failing

  1) trim
       trims all ES5 whitespace:

      AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
+ actual - expected

+ '"᠎             　  Hello, World!\\t\\n\\v\\f\\r   ᠎"'
- '"Hello, World!"'
      + expected - actual

      -"᠎             　  Hello, World!\t\n\v\f\r   ᠎"
      +"Hello, World!"
      
      at strictEqual (test/shared/eq.js:8:10)
      at Context.eq (test/trim.js:13:5)
      at processImmediate (internal/timers.js:456:21)
```

**Comment by adispring** [comment]
I think Ramda should add a CI/CD procedure,  so that only those Pull Requests passed CI/CD's unit test can be merged.

**Comment by adispring** [comment]
I found Ramda have a setting file: .github/workflows/build.yaml, but it seems not working now.

**Comment by CrossEye** [comment]
Agreed.  I would like to get 0.28 out the door in the next week and then 1.0 within two months.  Then I want to revisit all the infrastructure, build, test, etc.

**Comment by adispring** [comment]
> Agreed. I would like to get 0.28 out the door in the next week and then 1.0 within two months. Then I want to revisit all the infrastructure, build, test, etc.

Great. If there is anything I can do, let me know. I'll do my best to help.

**Review by CrossEye — COMMENTED** [review]
LGTM

**Review by CrossEye — DISMISSED** [review]
LGTM

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 name: Build
 
-on: [push]
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```
Comment (customcommander) [inline .github/workflows/build.yml:9]: @adispring I already addressed this in #3206. Would you mind abandoning that change?

**Comment by customcommander** [comment]
Your build is failing because of linting errors which have been fixed in #3206. If you rebase your branch against `master` it should be fixed.

**Comment by adispring** [comment]
> Your build is failing because of linting errors which have been fixed in #3206. If you rebase your branch against `master` it should be fixed.

Done.

**Review by customcommander — APPROVED** [review]

**Comment by customcommander** [comment]
@adispring I think you only reverted the change to the GH workflow so this should be good to go as all other changes were previously approved already I believe. I'm happy for this to be merged.

**Pull request merged**